bindings/pyroot/pythonizations/python/ROOT/_pythonization/_rdataframe.py

```diff
@@ -388,6 +389,17 @@ def __call__(self, *args):
 
         return res
 
+def _PyFilter(rdf, func, funcargs = {}, filter_name = ""):
+    """
+    Filters the RDF according to the function or string provided.
+    """
+    if isinstance(func, str): # If string argument is passed. Invoke the Orginal Filters.
+        return rdf._OriginalFilter(func)
```

Comment:
If args[0] is a string, you still need to pass the rest of the arguments to the original filter. The user could have specified a second argument for the name of the filter. Or they could have passed some additional argument that is wrong, in that case the original filter will error out accordingly. So this would be `return rdf._OriginalFilter(*args)`.